src/application/validation/validators/password-validation.ts

```diff
@@ -10,6 +10,9 @@ export class PasswordValidation implements Validation {
   ) {}
 
   validate (input: any): Error | void {
+    if (typeof input[this.fieldName] !== 'string') {
+      return new InvalidParamError(this.fieldName)
+    }
     const isValid = this.validator.isValid(input[this.fieldName])
     if (!isValid) {
       if (this.customError) {
```

Comment:
About custom error, I think that this is important to return more descriptive error for caller. On body validation an "InvalidParamError" is too generic, just means that the param is not an string or is not provided. For example, password validation case we need show to caller that params are correctly sended but they missmatch in values.